src/main/java/com/ericsson/eiffel/remrem/semantics/eiffelSchemas/EiffelRepoJsonSchema.java

```diff
@@ -0,0 +1,73 @@
+package com.ericsson.eiffel.remrem.semantics.eiffelSchemas;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import org.apache.commons.io.FileUtils;
+
+/**
+ * This class is used to Iterate over the Eiffel schemas that are cloned from
+ * Eiffel repository
+ * 
+ * @author xdurvak
+ *
+ */
+public class EiffelRepoJsonSchema {
+	private HashMap<String, File> jsonFiles;
+	private static ArrayList<String> jsonFileNames;
+
+	/**
+	 * This method is used to read Eiffel Schemas that are cloned from Eiffel
+	 * Repo
+	 * 
+	 * @param localPath
+	 *            - The Cloned Eiffel Repo path is sent as a parameter
+	 */
+	public void readEiffelRepoSchemas(File localPath) {
+		try {
+			FileUtils.cleanDirectory(new File(EiffelConstants.USER_DIR + "\\" + EiffelConstants.INPUT_EIFFEL_SCHEMAS));
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		jsonFileNames = new ArrayList<String>();
+		jsonFiles = new HashMap<String, File>();
+		String filePath = localPath + EiffelConstants.SCHEMA_LOCATION;
+		File file = new File(filePath);
+		if (file.isDirectory()) {
+			loadEiffelSchemas(filePath, "");
+		}
+		Iterator<String> iter = jsonFileNames.iterator();
+		while (iter.hasNext()) {
+			String filename = iter.next();
+			File jsonFile = jsonFiles.get(filename);
+			new ChangeJsonSchemaProperty().modifyJsonFileContent(jsonFile, filename, jsonFileNames);
```

Comment:
The modification of the schemas should be moved out of this method to its own method. That should be called from the main method.